### PR TITLE
Initial checkin for Cassandra benchmark tests for CentOS 6/7

### PR DESCRIPTION
**Pull request by khangngu**
Hi Christopher,
Please review and suggest how to add this test to the nightly automation test run.  Please read the README file for instructions on how to run the test.
Thanks,
-khang-

### CODE REVIEW

File: run-cassandra-centos.sh [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+PROFILE=
+PACKAGE=
+IMAGENAME=
+VERSION=
+UUID=
+NAME=
+IP1=
+IP2=
+TAG="test-instance=true"
+SCRIPT=$PWD/userscript.sh
```
Comment (chorrell) [inline run-cassandra-centos.sh]: I think this and all the relevant code for METADATAFILE can be removed. It was used for testing that user-script and user-data works but I'm assuming the cassandra tests don't rely on either of those.

File: run-cassandra-centos.sh [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    fi
+    
+    echo "    Public:  $PUBLIC_NETWORK"
+    echo "    Private: $PRIVATE_NETWORK"
+    echo ""
+}
+
+create_instance() {
+    echo "Creating instance ${1}:"
+    triton -p ${PROFILE} instance create -w -n $1 -N $PUBLIC_NETWORK -N $PRIVATE_NETWORK -t $TAG --script=$SCRIPT -M $METADATAFILE $2 $3
```
Comment (chorrell) [inline run-cassandra-centos.sh]: Per my previous comment, I think this part and all the supporting code in the script can be removed:

```
--script=$SCRIPT -M $METADATAFILE
```

File: run-cassandra-centos.sh [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+}
+
+cleanup() {
+    echo "Cleaning up."
+    rm -rf userscript.sh
+    rm -rf user-data
```
Comment (chorrell) [inline run-cassandra-centos.sh]: See previous comments about removing:

```
SCRIPT=$PWD/userscript.sh
METADATAFILE="user-data=$PWD/user-data"
```

Comment (khangngu) [inline run-cassandra-centos.sh]: Thanks Christopher.  Will update the run-cassandra-centos.sh per your comments.

**Comment by khangngu** [comment]
Close this request, and submit a new one.

**Pull request closed without merging**